src/Constants.js

```diff
@@ -94,26 +94,26 @@ exports.LastChanceOptions = {
 };
 
 /**
- * The giveaways manager options
+ * The giveaways manager options.
  * @typedef GiveawaysManagerOptions
  *
  * @property {string} [storage='./giveaways.json'] The storage path for the giveaways.
  * @property {number} [updateCountdownEvery=5000] The giveaway update interval (in ms).
  * @property {number} [endedGiveawaysLifetime=null] The time (in ms) after which a ended giveaway should get deleted from the DB. ⚠ Giveaways deleted from the DB cannot get rerolled anymore!
```

Comment:
```suggestion
 * @property {number} [updateCountdownEvery=5000] The giveaway update interval in milliseconds.
 * @property {number} [endedGiveawaysLifetime=null] The number of milliseconds after which ended giveaways should get deleted from the DB. ⚠ Giveaways deleted from the DB cannot get rerolled anymore!
```
Since you wrote it out for "threshold" too